ocs_ci/deployment/rosa.py

```diff
@@ -317,7 +317,7 @@ def host_network_update(self):
         security_group = self.aws.ec2_resource.SecurityGroup(sg_id)
         # The ports are not 100 % clear yet. Taken from doc:
         # https://docs.google.com/document/d/1RM8tmMbvnJcOZFdsqbCl9RvHXBv5K2ZI6ziQ-YTloGk/edit#
-        machine_cidr = config.ENV_DATA.get("machine-cidr", "10.0.0.0/16")
+        machine_cidr = config.ENV_DATA.get("machine_cidr", "10.0.0.0/16")
```

Comment:
If you're changing `machine-cidr` to `machine_cidr`, then I suppose you'll need to change it also in conf/deployment/rosa/private_link_deployment.yaml

Reply:
no need of defining default cidr in the private link conf as it is considered in a code. updated private_link_deployment.yaml conf